README.md

```diff
@@ -155,6 +155,20 @@ Mixer suited for audio signals.
 	- `In3`
 	- `In4`
 	- `Out`
+	
+### Decimator
```

Comment:
I have a loose rule to keep the "short names" of modules to be of max 8 characters in length. my aim is these names should fit on a small (norns) screen. Can you change the name to accommodate for that? :)

As an aside: A while back I also started going through input/output/parameter names to make them follow the same rule, but after changing all `FrequencyModulation` instances to `FM` I haven't spend time to complete this. I now created a github issue to handle it (https://github.com/antonhornquist/r/issues/2). It is better be made in one go and I'll probably do this pretty soon since these are breaking changes.

... in the long run, of course, a completely modular synth script which dynamically controls R from norns is totally feasible. Then short names are good for the UI, that's the idea.

Reply:
Gotcha - `Decimate` would be right at 8, or I could go shorter with `Deci` or `Crush` or something

Reply:
I'll let you decide but Decimate is fine.